app/modules/label_extractors/shared/cosmic_dawn_and_euclid.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require_relative '../base_extractor'
+
+module LabelExtractors
+  module Shared
+    class CosmicDawnAndEuclid < BaseExtractor
```

Comment:
I was just trying to group similar logic as much as possible but i get this might be an overkill at the moment. I've removed the shared class and overridden the necessary method on the Decals class

Reply:
agreed the simpler path is better at this early-ish stage; glad the change was made.